src/test/java/org/folio/search/controller/EsInstanceToInventoryInstanceIT.java

```diff
@@ -34,4 +34,17 @@ void responseContainsOnlyBasicInstanceProperties() throws Exception {
     assertThat(actual.getContributors(), is(expected.getContributors()));
     assertThat(actual.getPublication(), is(expected.getPublication()));
   }
+
+  @Test
+  void responseContainsAllInstanceProperties() throws Exception {
+    final var expected = getSemanticWeb();
+    final var actualJson = mockMvc.perform(get(searchInstancesByQuery("id=={value}&expandAll=true"), expected.getId())
```

Comment:
the same with final word, is this some kind code style?

Reply:
Nope, just my personal preference.

Reply:
It looks redundant here and in the other occurrences, it makes sense in code only if you really want to show that this field value should not be changed, otherwise - it is overloading of code by using redundant modifiers, other developers know, that there is no need to rewrite value of method parameter

Reply:
Back to the old days, the best practices mentioned that `final` helps to indicate coder's intention and also helps with potential complier optimization. But for a very local scoped variables, it is not necessary. Simpler/cleaner code style is more preferred in my view.

Reply:
My intention was that a `final` variable must be never changed, even by mistake.

In this case the `expected` value should be constant and never changed. But i'm fine to not put it, if it makes code clearer.